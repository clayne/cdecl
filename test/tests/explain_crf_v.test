EXPECTED_EXIT=0
cdecl -xc++ <<END
explain void f() const &
END
