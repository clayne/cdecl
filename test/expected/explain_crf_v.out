declare f as constant reference function returning void
